### PR TITLE
Added default icon support for nullscape datapack

### PR DESCRIPTION
**Pull request by PssbleTrngle**
## 🏗 Pull Request

- [x] Used a clear / meaningful title for this pull request.
- [x] Committed the changes and tested them in your userscript manager.
- [x] Tested all entrypoints for cross-browser support (`src` folder).
- [x] Have read the **Contributing** part of the **README**.
- [x] Passed `npm test`.
- [x] I noted my changes in the pull request body and/or changelog.

#### 👷🏾‍♀️ Fixes

- Map Icons for the custom `end_island` dimension added by the [Nullscape](https://www.planetminecraft.com/data-pack/nullscape/) datapack.

#### Description

Maps of the nullscape end island dimension use the overworld icons, instead of the end icons.

#### What changes have you made?

- Some css rules have been adapted to include the `nullscape_end_island` maps.

#### What tests have you completed?

- [x] Tested this in Chrome 100.
- [x] Tested this in Edge 99.
- [x] Tested this in Firefox 98.
- [x] Tested this in Opera 83.
- [x] Tested this in Electron 15.
- [x] Tested this in Bukkit 1.17.1.

### CODE REVIEW

File: src/main.css [hunk 1]
```diff
@@ -286,13 +286,16 @@ body {
 
 .dynmap .leaflet-control-container .leaflet-minimap-container[class*="the_end"] .world .item .maptype,
 .dynmap .leaflet-control-container .leaflet-minimap-container[class*="DIM1"] .world .item .maptype,
+.dynmap .leaflet-control-container .leaflet-minimap-container[class*="nullscape_end_island"] .world .item .maptype,
```
Comment (SNDST00M) [inline src/main.css:289]: Great! I'll note Nullscape support in the changelog, but a more generic solution would be good for the MC community here:
```suggestion
.dynmap .leaflet-control-container .leaflet-minimap-container:is([class^="end_"], [class*="_end_"]) .world .item .maptype,
```

**Review by SNDST00M — CHANGES_REQUESTED** [review]

**Comment by PssbleTrngle** [comment]
I got a comment that someone requested a change but I'm not seeing anything here now and I'm slightly worrying if I'm going insane

**Comment by SNDST00M** [comment]
I actually did suggest a more generic solution, but wanted to wait until tomorrow night when my laptop's repaired

I have however forgotten the selectors I was trying to target in the first place 😅

**Comment by SNDST00M** [comment]
I think what you did is best :) will look at this and custom favicon support tomorrow

**Comment by PssbleTrngle** [comment]
Some sort of custom icon support would be good too.
Even only using the default icons already provided for other dimensions (like using `the_end` icons for nullscape)

**Pull request merged**